### ISSUE TITLE
[BUG] waagent swapoffs my swapfile at boot.

### ISSUE THREAD

**Issue by sseiichi** [Azure/WALinuxAgent#1733]
I need additional swapfile, for example, /swapfile, so I added the following line to /etc/fstab.
```
/swapfile       swap            swap    default                         0 0
```

This works OK.

However, when I set the following option in /etc/waagent.conf, my /swapfile is not mounted at boot.
```
# Create and use swapfile on resource disk.
ResourceDisk.EnableSwap=y
```

Does waagent swapoff all the swapfiles at boot? I want to use both my swapfile and waagent's swapfile.

 - Distro and Version: Gentoo Linux
 - WALinuxAgent version:
WALinuxAgent-2.2.40 running on gentoo 2.6
Python: 3.6.9
Goal state agent: 2.2.45

**Comment by trstringer** [comment]
Thanks for bringing this to our attention! You are correct, it looks like `swapoff -a` is called if there is an existing swapfile of different size. I've submitted a fix for this in #1739.

**Comment by sseiichi** [comment]
Thanks a lot for your investigation and fixing.

**Comment by trstringer** [comment]
Thank *you* for bringing this to our attention! This has been fixed now in #1739.
